WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -188,24 +190,74 @@ extension NSNotification.Name {
         }
     }
 
-    func cacheUnlocalizedSitePlans() {
-        guard !WordPressComLanguageDatabase().deviceLanguage.slug.hasPrefix("en") else {
-            // Don't fetch if its already "en".
-            return
-        }
-
+    func cacheUnlocalizedSitePlans(accountService: AccountService? = nil, planService: PlanService? = nil) {
         let context = ContextManager.shared.mainContext
-        let accountService = AccountService(managedObjectContext: context)
+        let accountService = accountService ?? AccountService(managedObjectContext: context)
         guard let account = accountService.defaultWordPressComAccount() else {
             return
         }
 
-        let planService = PlanService(managedObjectContext: context)
+        let planService = planService ?? PlanService(managedObjectContext: context)
         planService.getAllSitesNonLocalizedPlanDescriptionsForAccount(account, success: { plans in
             self.sitePlansCache = plans
         }, failure: { error in })
     }
 
+    /// Retrieves the highest priority plan, if it exists
+    /// - Returns: the highest priority plan found, or an empty string if none was foundq
+    private func getHighestPriorityPlan() -> String {
```

Comment:
Can this be a static method to follow the pattern of the other methods? Also, there is a private extension this might could be moved into.